app/javascript/actions/gameActions.js

```diff
@@ -1,15 +1,16 @@
-import axios from 'axios';
-import 'dotenv/config';
+import axios from "axios";
+import "dotenv/config";
```

Comment:
Do we still need `dotenv` if we are not providing base URL from the `.env` file?